### DISCUSSION TITLE
Fix, you do not have access to Chrome Web Store

### DISCUSSION THREAD

**Discussion by phillipbruh** [3kh0/ext-remover#41] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
If you want to unblock webstore make sure when you goto it it says this

We are sorry, but you do not have access to Chrome Web Store. Please contact your Organization Administrator for access.

make sure you have 2 google accounts your school and your personal then goto this link
https://chrome.google.com/webstore?authuser=1

**Comment by 3kh0** [comment, marked as answer]
Thank you for the fix!

**Comment by Aka-but-nice** [comment]
Why tf would we have access to a personal account?
